build/pipelines/release.yml

```diff
@@ -265,6 +269,231 @@ jobs:
+        versionListDownload: $(TerminalInternalPackageVersion)
+    - task: TouchdownBuildTask@1
+      displayName: Download Localization Files
+      inputs:
+        teamId: 7105
+        authId: $(TouchdownAppId)
+        authKey: $(TouchdownAppKey)
+        resourceFilePath: >-
+          src\cascadia\TerminalApp\Resources\en-US\Resources.resw
+
+          src\cascadia\TerminalControl\Resources\en-US\Resources.resw
+
+          src\cascadia\TerminalConnection\Resources\en-US\Resources.resw
+
+          src\cascadia\TerminalSettingsModel\Resources\en-US\Resources.resw
+
+          src\cascadia\TerminalSettingsEditor\Resources\en-US\Resources.resw
+
+          src\cascadia\WindowsTerminalUniversal\Resources\en-US\Resources.resw
+
+          src\cascadia\CascadiaPackage\Resources\en-US\Resources.resw
+        appendRelativeDir: true
+        localizationTarget: false
+        pseudoSetting: Included
+    - task: PowerShell@2
+      displayName: Move Loc files one level up
+      inputs:
+        targetType: inline
+        script: >-
+          $Files = Get-ChildItem . -R -Filter 'Resources.resw' | ? FullName -Like '*en-US\*\Resources.resw'
+
+          $Files | % { Move-Item -Verbose $_.Directory $_.Directory.Parent.Parent -EA:Ignore }
+        pwsh: true
+
+    # 1ES Component Governance onboarding (Detects open source components). See https://docs.opensource.microsoft.com/tools/cg.html
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+      displayName: Component Detection
+      
+    # # PREfast and PoliCheck need Node. Install that first.
+    - task: NodeTool@0
+
+    # !!! NOTE !!! Run PREfast first. Some of the other tasks are going to run on a completed build.
+    # PREfast is going to build the code as a part of its analysis and the generated sources
+    # and output binaries will be sufficient for the rest of the analysis.
+    # If you disable this, the other tasks won't likely work. You would have to add a build
+    # step instead that builds the code normally before calling them.
+    # Also... PREfast will rebuild anyway so that's why we're not running a normal build first.
+    # Waste of time to build twice.
+    # PREfast. See https://www.1eswiki.com/wiki/SDL_Native_Rules_Build_Task 
+
+    # The following 1ES tasks all operate completely differently and have a different syntax for usage.
+    # Most notable is every one of them has a different way of excluding things.
+    # Go see their 1eswiki.com pages to figure out how to exclude things.
+    # When writing exclusions, try to make them narrow so when new projects/binaries are added, they
+    # cause an error here and have to be explicitly pulled out. Don't write an exclusion so broad
+    # that it will catch other new stuff.
+
+    # https://www.1eswiki.com/wiki/PREfast_Build_Task
+    # Builds the project with C/C++ static analysis tools to find coding flaws and vulnerabilities

```

Comment:
Is this going to be a problem for non-release builds? Like, if we introduce a flaw/vulnerability before release and we don't find out until we're actually trying to run a release build?

tl;dr: should we also run this as a part of the regular CI pipeline?

Reply:
I thought about that but it's over an hour to run this whole set of compliance tasks end to end. So we likely need to discuss in team meeting how we're going to handle that. I was going to turn it on in Release for now and then work on rolling it out further. Kinda like how we need to roll those UIA tests out further...